### PR TITLE
setup: fix `launch` command on Windows

### PR DESCRIPTION
**Pull request by benoit-pierre**
Same issue as with `dist_main.main`.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -121,7 +121,12 @@ def run(self):
             python_path = os.environ.get('PYTHONPATH', '').split(os.pathsep)
             python_path.insert(0, sys.path[0])
             os.environ['PYTHONPATH'] = os.pathsep.join(python_path)
-            os.execv(sys.executable, [sys.executable, '-m', 'plover.main'] + self.args)
+            cmd = [sys.executable, '-m', 'plover.main'] + self.args
+            if sys.platform.startswith('win32'):
+                # Workaround https://bugs.python.org/issue19066
```
Comment (user202729) [inline setup.py:126]:
```suggestion
                # Workaround https://bugs.python.org/issue19124
```

Comment (benoit-pierre) [reply setup.py:126]: The problem is spaces in the executable directory, not a background execution issue.

Comment (user202729) [reply setup.py:126]: Ah, I was confused. (Plover's workaround for background execution was the issue I got when I tried to execute plover_console_ui on Windows)

Comment (user202729) [reply setup.py:126]: Looks like the "correct" duplicate is https://bugs.python.org/issue436259 which is closed (fixed?)?

Comment (benoit-pierre) [reply setup.py:126]: Yeah ¯\\_(ツ)_/¯... That's with Python 3.8, I've not tested back with 3.6.

**Review by morinted — APPROVED** [review]

**Pull request merged**